PayPal/build.gradle

```diff
@@ -32,6 +32,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation project(':BraintreeDataCollector')
 
+    implementation('com.paypal.checkout:android-sdk:0.1.0') {
+        exclude module: 'data-collector'
+    }
```

Comment:
This was fixed from the previous PR. This module was causing a duplicate class error.